packages/tools/src/tools/segmentation/strategies/fillCircle.ts

```diff
@@ -67,12 +66,13 @@ const initializeCircle = {
       segmentationImageData.getDimensions()
     );
 
-    segmentationVoxelManager.boundsIJK = boundsIJK;
-    imageVoxelManager.isInObject = createPointInEllipse({
+    operationData.isInObject = createPointInEllipse({
```

Comment:
Here is a spot where getting the object iterator as a complete options object which includes currently the is in object and the bounds ijk would allow us in the future to define it using more efficient maps, and as long as it is just an instance of the right type, then the type can be extended in the future to allow for improved definitions.  I believe this should be part of CS3D 2 as it fixes the types now, and then allows for better future extension.

Reply:
We have the options for the forEach, i added it to the pointInShape too